ui/src/tool-panel/switch/dialog/switch-edit-dialog.tsx

```diff
@@ -368,10 +363,10 @@ export const SwitchEditDialog = ({
                             errors={getVisibleErrorsByProp('name')}>
                             {conflictingSwitch && (
                                 <>
-                                    <div className={styles['switch-edit-dialog__alert']}>
-                                        {t('switch-dialog.name-in-use')}
+                                    <div className={styles['switch-edit-dialog__alert-color']}>
+                                        {conflictingSwitch.stateCategory === 'NOT_EXISTING' ? t('switch-dialog.name-in-use-deleted') : t('switch-dialog.name-in-use')}
```

Comment:
Päästääkös prettier oikeasti läpi näin pitkiä rivejä, vai onko mennyt pois päältä?